agenthub/SWE_agent/prompts.py

```diff
@@ -191,7 +192,7 @@ def MEMORY_FORMAT(act, obs):
 {unpack_dict(act, ["content"])}
 
 Output from Action:
-{unpack_dict(obs)}
+{unpack_dict(obs), ["content"]}
```

Comment:
Two comments:
1. This changes the program behavior, is that intentional?
2. If it is intentional, maybe there's a mis-placed parenthesis?

```suggestion
{unpack_dict(obs, ["content"])}
```

Reply:
Re: 1. You're right, I just verified, this changed the previous behavior and I don't think it should. Fixed now.